Src/AutoFixture.xUnit.net.UnitTest/DelegatingFixture.cs

```diff
@@ -101,5 +102,7 @@ public object Create(object request, ISpecimenContext context)
         }
 
         internal Func<object, ISpecimenContext, object> OnCreate { get; set; }
+
+        internal Action<ICustomization> OnCustomize { get; set; }

```

Comment:
The method being implemented has the signature `public IFixture Customize(ICustomization customization)`. It returns an object.

The intention of all `DelegatingXyz` classes in the AutoFixture test code base is that they serve as general-purpose Test Doubles. Someone else may, in the future, need to configure the `Customize` method to return a particular object. That isn't going to be possible with the current design, and I'd think it'd be a shame if we make something like that too difficult in the future.

Granted, it makes writing the current test a bit more difficult, but it's more principled, and I don't think that the cost is _that_ big.

You could argue that we ought to apply the YAGNI principle and keep it as you already wrote it, and you'd have a good point. On the other hand, I've seen examples of code bases where a too unprincipled approach to test API design could lead to code that was hard to change. My scars are showing, I know :sweat_smile: 

I can see that I failed to notice this in [your previous pull request](https://github.com/AutoFixture/AutoFixture/pull/590), but I'll go and address that myself.